1/bump_version.py

```diff
@@ -188,6 +189,36 @@ def update_version_in_file(version_file_path: str, new_version: str):
             version_file.write(new_content)
 
 
+def add_ticket(changelog_path, new_version):
+    ticket_base_url = os.getenv('PLUGIN_TICKET_URL')
+    if not ticket_base_url:
+        print(f"Not adding the ticket to changelog, because no url (PLUGIN_TICKET_URL) to ticketing was provided.")
+
+    source_branch = os.getenv("DRONE_SOURCE_BRANCH")
+    print(f"The source_branch is {source_branch}")
+    if not source_branch:
+        return
+
+    # Get key of the ticket from the source branch name
+    match = re.search(os.getenv('PLUGIN_TICKET_PATTERN'), source_branch, re.IGNORECASE)
```

Comment:
I think there should be a check for this plugin parameter beforehand, even before checking the source branch, to properly raise the issue if not provided.